Gemfile

```diff
@@ -21,6 +21,7 @@ gem "puma", "~> 3.4"
 gem "whenever", "~> 0.9"
 gem "validates_email_format_of", "~> 1.6"
 gem "redcarpet"
+gem "jbuilder", "~> 2.4"
```

Comment:
I'm not 100% convinced we should use jbuilder for this. It feels like an extra dependency that we can also code out ourselves.